src/Symfony/Component/HttpClient/Response/ResponseTrait.php

```diff
@@ -299,6 +317,16 @@ public static function stream(iterable $responses, float $timeout = null): \Gene
                         $isTimeout = false;
 
                         if (\is_string($chunk = array_shift($multi->handlesActivity[$j]))) {
+                            if (null !== $response->inflate && false === $chunk = @inflate_add($response->inflate, $chunk)) {
+                                $multi->handlesActivity[$j] = [null, new TransportException('Error while processing content unencoding.')];
+                                continue;
+                            }
+
+                            if ('' !== $chunk && null !== $response->content && \strlen($chunk) !== fwrite($response->content, $chunk)) {
+                                $multi->handlesActivity[$j] = [null, new TransportException('Failed writing %d bytes to the response buffer.', \strlen($chunk))];
```

Comment:
@nicolas-grekas I think `sprintf` is supposed to be here.

Reply:
correct :) would you mind sending a PR?

Reply:
Done #35250